src/main/java/randoop/main/GenTests.java

```diff
@@ -319,6 +318,11 @@ public boolean handle(String[] args) {
 
     List<TypedOperation> operations = operationModel.getOperations();
 
+    if (GenInputsAbstract.enable_bloodhound) {
+      // Copy the list of methods under test into Bloodhound's internal list.
+      Bloodhound.instance.initBloodhound(operations);
```

Comment:
Why is Bloodhound an singleton that is created when Randoop is loaded, but then side-effected later?
This is a confusing design pattern.  I think it should be created with the set of operations, which should not be side-effected later.
Then the logic of `initBloodhound` can be moved into the Bloodhound constructor.
Even if you want to have an instance, the instance could be created here.

The same comment applies to `CoverageTracker`:  put the logic of `setClassesUnderTest` in the constructor.
Can CoverageTracker be a field of Bloodhound?  Encapsulating the logic within Bloodhound would keep Randoop simpler and more encapsulated, if it is possible.

Also, can you combine the two occurrences of 
```
    if (GenInputsAbstract.enable_bloodhound) {
```
?  Putting the logic together will be cleaner, I think.
Or is that impossible because `createModel` loads the classes?  In that case, document it.